action.yml

```diff
@@ -41,18 +41,22 @@ inputs:
         Passing multiple package names is allowed.
         Package names can be separated by any whitespace character.
     required: true
-  source-ros-binary-installation:
+  target-ros1-distro:
```

Comment:
What's the default behavior for these if not specified?

Reply:
Either target-ros1-distro or target-ros2-distro needs to be specified or it will return an error.